integrations/postgresql-integration.mdx

```diff
@@ -3,7 +3,7 @@ title: "PostgreSQL"
 description: PostgreSQL Integration - Setup & Installation Docs | Middleware
 ---
 
-Collects PostgreSQL performance and health metrics
+The PostgreSQL Integration allows users to ingest and monitor the performance and health metrics of their object-relational database management system (ORDBMS) data. This integration is useful for tracking the total number of blocks read into memory, identifying disk space utilization per table, tallying insert operations and rollbacks triggered, and much more.
```

Comment:
In this page we see "users to...their" where in the oracledb we see "your". I think we should go with "you/your" pronouns as a more direct approach than "users/their". Make sure we're consistent across all docs as well por favor

Reply:
(Start with just integrations right now if we need to)